### ISSUE TITLE
onLoad test fails

### ISSUE THREAD

**Issue by jonschlinkert** [jonschlinkert/template-toc#1]
@doowb, not sure why but the `.onLoad()` [test fails](https://github.com/jonschlinkert/template-toc/blob/master/test.js#L47). Does anything obvious stand out that I missed?

**Comment by doowb** [comment]
`onLoad` middleware runs right after a template is loaded. You're loading `fixtures.md` and `doc.md` in the `beforeEach` method which runs before you're adding the `onLoad` middleware.

You could reload the templates for that test after adding the middleware to make sure it runs.

**Comment by jonschlinkert** [comment]
good point, I spend so much time in the tests I always forget that `beforEach()` function is even there
